src/foam/u2/dialog/Popup.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+foam.CLASS({
```

Comment:
I can add documentation.

As noted earlier, this is trying to do something completely different from the other `PopupView`.

`foam.u2.PopupView` is a parent-relative popup. `foam.u2.dialog.Popup` is a container for a whole-screen modal dialog. I don't think this is an example of FOAM1-style duplication. I thought we were aiming for a more Unixy model anyway, of well-encapsulated views that do exactly one thing?